post.md

```diff
@@ -1,22 +1,22 @@
 For a while I've wanted to explore Suspense.
-I read a react blog post way back in 2019 about
+I read a React blog post way back in 2019 about
 [building great UX with React Suspense](https://legacy.reactjs.org/blog/2019/11/06/building-great-user-experiences-with-concurrent-mode-and-suspense.html).
-I remember it giving me a real aha moment.
-The description of the waterfall approach to loading data vs. loading data as you render really resonated with performance problems we'd had in a project I was working with at the time.
-Suspense though was experimental in react 16; the only thing that it was supported with was `Lazy` components,
-and I never like using experimental API's in production
+I remember it giving me a real "aha!" moment.
+The description of the waterfall approach to loading data vs loading data as you render really resonated due to performance problems we'd had in a project I was working with at the time.
```

Comment:
much, much further down the post you give a bit more description of what you mean by a waterfall approach to loading data. I think that at least needs to be up here instead, or you should give an example of what you mean:

"You don’t want to render a component, wait for the data to load, only to render a child which also needs to wait for data to load"